execution/evm/vm.go

```diff
@@ -493,6 +493,52 @@ func (vm *VM) call(caller acm.Account, callee acm.MutableAccount, code, input []
 			stack.Push64(int64(res))
 			vm.Debugf(" => 0x%X\n", res)
 
+		case SHL: //0x1B
+			shift, value := stack.PopBigInt(), stack.PopBigInt()
```

Comment:
`value` shadows the `value` argument to `call`. Can we change it to something else, I suggest just `x` as used in other arithmetic functions.